test/Microsoft.SqlTools.Migration.IntegrationTests/Microsoft.SqlTools.Migration.IntegrationTests.csproj

```diff
@@ -24,6 +24,7 @@
   </Choose>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.TestPlatform" VersionOverride="14.0.0" />
```

Comment:
Why you needed to add this? is it required?

Reply:
Yes , there are 2 Assert functions being used from this.